DocsByReflection.UnitTests.Stubs/DocsByReflection.UnitTests.Stubs.csproj

```diff
@@ -1,64 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>10.0.0</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{4514ACEE-9543-4A15-B0AE-00B7D58587DB}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>DocsByReflection.UnitTests.Stubs</RootNamespace>
-    <AssemblyName>DocsByReflection.UnitTests.Stubs</AssemblyName>
+    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
I don't know wheter we should use TargetFrameworks tag to suppot .NET Framework 4.6.2 (net462) too? Or just document that who still in .NET Framework should use:

```shell
install-package DocsByReflection -Version 1.0.12.20
```

Reply:
Yeah we could do multi-targeting.